modules/monitoring-support-version-matrix-for-monitoring-components.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="support-version-matrix-for-monitoring-components_{context}"]
+= Support version matrix for monitoring components
+
+The following matrix contains information about versions of monitoring components for {product-title} 4.12 and later releases:
+
+|===
+|{product-title} 9+|Component Versions
+
+|*Version* |*Prometheus Operator* |*Prometheus*  |*Prometheus Adapter* |*Metrics Server* (Technology Preview) |*Alertmanager* |*kube-state-metrics agent* |*monitoring-plugin* |*node-exporter agent* |*Thanos*
```

Comment:
So I'll admit that this is a smart way of conveying this information, although this _sort of_ seems to go against our [SSG's guidance on tables](https://redhat-documentation.github.io/supplementary-style-guide/#accessibility-tables), where the second bullet point says to avoid tables with irregular headers. If you click on the link in the sub-bullet point for examples, one of the examples on that page include tables with two tier headers.

I think in this case, you could probably just make one header row and ensure that everything has "version" at the end of it:
```suggestion
|*{product-title} version* |*Prometheus Operator version* |*Prometheus version*  |*Prometheus Adapter version* |*Metrics Server version* (Technology Preview) |*Alertmanager version* |*kube-state-metrics agent version* |*monitoring-plugin version* |*node-exporter agent version* |*Thanos version*
```

Alternatively, if that would make some of the cells too packed with text, you could consider using a table title like `.{product-title} and component versions` in between lines 10 and 11, and then make the table headers like this:
```suggestion
|*{product-title}* |*Prometheus Operator* |*Prometheus*  |*Prometheus Adapter* |*Metrics Server* (Technology Preview) |*Alertmanager* |*kube-state-metrics agent* |*monitoring-plugin* |*node-exporter agent* |*Thanos*
```

Reply:
Hello, thank you for letting me know about this, I really like the second suggestion, so I will apply that one :+1: